app/controllers/sms/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Sms::UsersController < Sms::BaseController
+  def update
+    user = User.find_by_phone(params["From"])
+    if (params["Body"].upcase[0..1] == "NO" || params["Body"].upcase[0..10] == "UNSUBSCRIBE") && user
```

Comment:
this totally works - i was thinking they would respond "YES" or something to confirm they're signed up rather than the opposite where they say no if they don't want to join. this way they'll confirm positively? 

what do you think? i also think this is totally fine!